bin/pulumilocal

```diff
@@ -41,7 +49,7 @@ def generate_service_endpoints(args) -> Dict:
     if args.cwd:
         cmd_args.extend(("--cwd", args.cwd))
     try:
-        sp = subprocess.run(executable=PULUMI_CMD, args=cmd_args, env=os.environ, stdout=subprocess.PIPE, check=True)
+        sp = subprocess.run(executable=PULUMI_CMD, args=cmd_args, env=os.environ, check=True, bufsize=0, capture_output=True)
```

Comment:
q: this argument is not supported on python <3.7, I assume we are ok with that?

Reply:
Officially we're supporting >3.8